tests/code/test_userManagement.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from app.logic.userManagement import addCeltsAdmin, removeCeltsAdmin,addCeltsStudentStaff, removeCeltsStudentStaff, changeCurrentTerm
+from app.models.user import User
+from peewee import DoesNotExist
+from flask import g
+@pytest.mark.integration
+def test_modifyCeltsAdmin():
+    user = "agliullovak"
+    current_user = User.get(User.username == user)
+    assert current_user.isCeltsAdmin == False
+    addCeltsAdmin(current_user)
+    current_user = User.get(User.username == user)
+    assert current_user.isCeltsAdmin == True
+    removeCeltsAdmin(current_user)
+    current_user = User.get(User.username == user)
+    assert current_user.isCeltsAdmin == False
+
+    with pytest.raises(DoesNotExist):
+        addCeltsAdmin("blahbah")
+    with pytest.raises(DoesNotExist):
+        addCeltsAdmin("ksgvoidsid;")
+
+def test_modifyCeltsStudentStaff():
+    user = "mupotsal"
+    current_user = User.get(User.username == user)
+    assert current_user.isCeltsAdmin == False
+    addCeltsStudentStaff(current_user)
+    current_user = User.get(User.username == user)
+    assert current_user.isCeltsAdmin == True
+    removeCeltsStudentStaff(current_user)
+    current_user = User.get(User.username == user)
+    assert current_user.isCeltsAdmin == False
+
+    with pytest.raises(DoesNotExist):
+        addCeltsStudentStaff("asdf")
+    with pytest.raises(DoesNotExist):
+        removeCeltsStudentStaff("1234")
+
+def test_changeCurrentTerm():
+    # test via g.current_term
+    changeCurrentTerm(2)
+    assert g.current_term == 2
+    # test via isCurrentTerm
+    newTerm = changeCurrentTerm(1)
+    assert newTerm.isCurrentTerm
```

Comment:
I'd also test that g.current_term is different after the second call, and that the old current term doesn't have isCurrentTerm = True anymore.